srcpkgs/greetd/template

```diff
@@ -0,0 +1,50 @@
+# Template file for 'greetd'
+pkgname=greetd
+version=0.7.0
+revision=1
+build_style=cargo
+conf_files="/etc/greetd/greetd.conf"
+hostmakedepends="scdoc"
+makedepends="pam-devel"
+short_desc="Minimal and flexible login manager daemon"
+maintainer="cinerea0 <cinerea0@protonmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://git.sr.ht/~kennylevinsen/greetd"
+distfiles="https://git.sr.ht/~kennylevinsen/${pkgname}/archive/${version}.tar.gz"
+checksum=5229b7f00be40187de9df6240fc3b1f4012dd5e4777bb8661016cc7c2c105df7
+
+system_accounts="_greeter"
+_greeter_groups="video"
+
+pre_build() {
+	# Void accounts are prefixed with underscore
+	vsed -i "16s/greeter/_greeter/" config.toml
```

Comment:
IMO a patch or configure arg should be used, otherwise it's easy for this fix to get lost.